biobesu/cli.py

```diff
@@ -0,0 +1,21 @@
+#!/user/bin/env python3
+
+from argparse import ArgumentParser
+from argparse import RawTextHelpFormatter
+from biobesu.helper.generic import retrieve_entry_point
+from biobesu.helper.argument_parser import BiobesuParser
+
+
+def main():
+    # Load available suites.
+    suites = retrieve_entry_point('biobesu_suites')
+
+    # Defines global command line.
+    parser = BiobesuParser(formatter_class=RawTextHelpFormatter, add_help=False)
+    parser.add_argument("suite", help="the chosen benchmark suite:\n" + '\n'.join(suites))
```

Comment:
Good point. It seems that argparse uses `\n` itself as well in [its code](https://github.com/python/cpython/blob/3.9/Lib/argparse.py#L230).

I took the liberty of trying it out on my Windows 10 desktop (after installing python3.9 & pip), which resulted in the output in the screenshot when running it through the windows command prompt. I'm not sure why this is formatted correctly (as according to the documentation [you should be right](https://docs.python.org/3/library/os.html#os.linesep)). Perhaps Python already makes this Windows-compatible?
![Schermopname (49)](https://user-images.githubusercontent.com/7658246/111296546-e9053080-864c-11eb-8be9-8aed4f6fdd6e.png)

Long story short: I suggest keeping the parts that are input for argparse similar to how argparse handles newlines itself. When we get reports of Windows users with the issue of a wrongly formatted usage, investigate the issue further.

Reply:
Right, that looks okay to me. I commented it because recently I used `\n` in a java application, which caused the application to fail on windows, so I looked for a python alternative, which apparently is `os.linesep`, although you only don't want to use it in text output. I had problems with this before. Especially when running in pycharm rather than directly from the commandline. If both works, I'd suggest using `os.linesep`, just to be sure.

Reply:
As `os.linesep` is not needed for files in text-mode, it might treat stdout/stderr the same way. Would explain the output I got on Windows. Furthermore [print()](https://docs.python.org/3/library/functions.html#print) also uses `\n` and not not `os.linesep`. So I would still suggest just keeping `\n` unless we get reports from Windows users regarding missing newlines.